devito/types/sparse.py

```diff
@@ -579,15 +584,15 @@ def _coordinate_indices(self):
                       for c, o, i in zip(self._coordinate_symbols, self.grid.origin,
                                          indices[:self.grid.dim])])
 
-    @cached_property
-    def _coordinate_bases(self):
+    def _coordinate_bases(self, field_offset):
```

Comment:
`memoized_meth` ?

what's the performance penalty if not cached ?

Reply:
This is only called at `interpolate` or `inject` which is not very frequent nor costly so I don't think there is much penalty here.